src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -694,37 +694,40 @@ bool Snippet::SnippetJitExecutor::optimizeExecDomain(std::vector<VectorDims>& in
 }
 
 void Snippet::SnippetJitExecutor::generate(const jit_snippets_compile_args* jcp) {
-    ov::pass::Manager pre_dialect;
-    pre_dialect.register_pass<ConvertToSwishCPU>();
+    using Manager = snippets::pass::Manager;
+    std::vector<Manager::PositionedPass> backend_passes;
+
+#define SNIPPETS_ADD_POS_PASS(POS, PASS, ...) \
```

Comment:
I think we should define this macros only in x86-64 architecture as in `CPU_REGISTER_PASS_X64`.